lib/node_modules/@stdlib/math/base/special/nonfibonacci/README.md

```diff
@@ -123,6 +123,82 @@ for ( i = 1; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/nonfibonacci.h"
+```
+
+#### stdlib_base_nonfibonacci( x )
+
+Computes the nth non-Fibonacci number.
+
+```c
+double out = stdlib_base_nonfibonacci( 1.0 );
+// returns 4
+
+out = stdlib_base_nonfibonacci( 2.0 );
+// returns 6
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+
+```c
+double stdlib_base_nonfibonacci( const double x );
+```
+
+</section>
+<!-- /.usage -->
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+<section class="notes">
+</section>
+<!-- /.notes -->
+<!-- C API usage examples. -->
+<section class="examples">
+### Examples
+```c
+#include "stdlib/math/base/special/nonfibonacci.h"
+#include <stdio.h>
+#include <stdlib.h>
+int main( void ) {
+    int i;
+    for ( i = 1; i < 12; i++ ) {
+        double result = stdlib_base_nonfibonacci( i );
+        printf( "x: %lf => result: %lf, frac: %lf\n",   i , result );
+    }
+}
+```
```

Comment:
````suggestion
</section>

<!-- /.usage -->

<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->

<section class="notes">

</section>

<!-- /.notes -->

<!-- C API usage examples. -->

<section class="examples">

### Examples

```c
#include "stdlib/math/base/special/nonfibonacci.h"
#include <stdio.h>
#include <stdlib.h>
int main( void ) {
    int i;
    for ( i = 1; i < 12; i++ ) {
        double result = stdlib_base_nonfibonacci( i );
        printf( "x: %lf => result: %lf, frac: %lf\n",   i , result );
    }
}
```
````